packages/squiggle-lang/src/rescript/Utility/Operation.res

```diff
@@ -37,22 +37,57 @@ module Convolution = {
     }
 }
 
+@genType
+module Error = {
+  @genType
+  type invalidOperationError =
```

Comment:
I think I'd probably just call this ``t``, and then the next function, ``toString``, but I don't care too much here.